src/Illuminate/Database/DatabaseManager.php

```diff
@@ -280,7 +280,12 @@ public function supportedDrivers()
      */
     public function availableDrivers()
     {
-        return array_intersect($this->supportedDrivers(), PDO::getAvailableDrivers());
+        return array_unique(
+            array_intersect(
+                $this->supportedDrivers(),
+                preg_replace('/^dblib$/', 'sqlsrv', PDO::getAvailableDrivers())
```

Comment:
Oooh also, it just occurred to me that an additional replacement (say there was some future driver called notsqlsrvdblib) would still not be returned in error because it would be replaced with notsqlsrvsqlsrv via str_replace. Sorry I should have thought of that.

Switched it to use str_replace.